packages/shared/lib/deepLinking.ts

```diff
@@ -1,6 +1,91 @@
-import { writable } from 'svelte/store'
+import { Unit } from '@iota/unit-converter';
+import { addError } from 'shared/lib/errors';
+import { writable } from 'svelte/store';
 
 /**
  * Indicates that a deep link request is active
  */
 export const deepLinkRequestActive = writable<boolean>(false)
+
+/**
+ * Parse a deep link for the app (iota://)
+ * @param {string} data Deep link data
```

Comment:
I think this needs just a little more TSDoc comment info for the function signature